babel.config.js

```diff
@@ -30,6 +30,7 @@ module.exports = function (api) {
         root: ['./src'],
       },
     ],
+    '@babel/plugin-transform-export-namespace-from',
```

Comment:
This was in order to get the expo modules to work nicely. See: https://github.com/dominicstop/react-native-ios-context-menu/issues/81#issuecomment-2123208410